fiftyone/operators/builtin.py

```diff
@@ -1939,18 +1939,18 @@ def resolve_input(self, ctx):
             ),
         )
 
-        # @todo infer this automatically from current App spaces
+        current_spaces = ctx.spaces.to_json(True) if ctx.spaces else None
```

Comment:
Ah, to clarify: here I was thinking that, now that we have `ctx.spaces`, we can just delete the `spaces` parameter entirely from `resolve_input()` and just use `ctx.spaces` in `execute()`.

Pasting in a spaces configuration as JSON was not something I wanted to support, it was just a placeholder until `ctx.spaces` was available 😄 

To support programmatic execution, we could still allow the `spaces` parameter to be optionally provided like so:

```py
def execute(self, ctx):
    spaces = ctx.params.get("spaces", None)

    if spaces is not None:
        spaces = _parse_spaces(spaces)
    else:
        spaces = ctx.spaces
```

Reply:
Ahh I see. Yah, that makes sense 😄

Reply:
Perhaps, for programmatic execution, I will leave the support. However, I'll remove from `resolve_input`